exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,27 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+
+function isNegative(number) {
+    // if(number < 0){
+
+    //   return true
+    // }
+    if (Math.sign(number) === -1) {
+        return true;
+    } else {
+        return false;
+    }
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+    if (number >= 0 && number <= 10) {
+        return true;
+    } else {
+        return false;
+    }
+}
```

Comment:
These work well as functions. Again, you can just return the result of the conditional statement.